client/src/components/IncomeAdder/IncomeAdder.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import {
+  Container,
+  TitleContainer,
+  Title,
+  Button,
+  IncomeForm,
+  InputContainer,
+  InputGroup,
+  Label,
+  Input,
+  FormButton,
+} from "./IncomeAdder.styles";
+import { MdOutlineCancel } from "react-icons/md";
+
+interface Props {
```

Comment:
Same as above.

Reply:
It's a simple task but I have a hard time naming these haha I've just changed it to IncomeAdderProps for now because I'm not sure what other props we may need to pass down later if any changes are made. Let me know if you have a suggestion on what best practices are for naming these interfaces!